src/components/CopyCodeModal/CopyCodeModal.js

```diff
@@ -36,35 +29,36 @@ class Modal extends Component {
     closePaletteCodeModal();
   }
 
-  onTabChange(content, tabName) {
-    this.setState({
-      currentText: content,
-      selectedTab: tabName,
-    });
+  onTabChange(selectedTab) {
+    this.setState({ selectedTab });
   }
 
   render() {
     const { contents, isVisibleCodeModal } = this.props;
-    const { selectedTab, currentText } = this.state;
+    const { selectedTab } = this.state;
+    const content = contents[selectedTab];
 
     const actions = [
       <FlatButton label="Cancel" secondary onClick={() => this.onClose()} />,
-      <CopyToClipboard text={currentText} onCopy={() => this.onClose(true)}>
+      <CopyToClipboard text={content} onCopy={() => this.onClose(true)}>
         <FlatButton label="Copy" />
       </CopyToClipboard>,
     ];
 
-    const tabs = contents.map(({ name, content }) => {
-      const selectedClass = name === selectedTab ? 'modal__tab--selected' : '';
+    const tabs = Object.keys(contents).map((tabName) => {
+      const className = {
+        modal__tab: true,
+        'modal__tab--selected': tabName === selectedTab,
+      };
       return (
         <div
-          key={name}
+          key={tabName}
           tabIndex={0}
           role="button"
-          className={`modal__tab ${selectedClass}`}
-          onClick={() => this.onTabChange(content, name)}
+          className={classNames(className)}
+          onClick={() => this.onTabChange(tabName)}
```

Comment:
Recently read that use arrow functions in render method is bad practise, coz every execution of `render` method creating new instance of this function and it`s bad for performance. And if u want to pass data in calling function u must create component like `Tab` pass this function in props and call it with data( tabName ) from this component. [Link](https://daveceddia.com/avoid-bind-when-passing-props/) on article.